File: src/main/java/com/sequenceiq/cloudbreak/service/usages/CloudbreakUsagesRetrievalService.java
package com.sequenceiq.cloudbreak.service.usages;

import java.util.List;

import com.sequenceiq.cloudbreak.domain.CbUsageFilterParameters;
import com.sequenceiq.cloudbreak.domain.CloudbreakUsage;

public interface CloudbreakUsagesRetrievalService {
    List<CloudbreakUsage> findUsagesFor(CbUsageFilterParameters params);
}

File: core/src/main/java/com/sequenceiq/cloudbreak/service/usages/CloudbreakUsagesRetrievalService.kt
package com.sequenceiq.cloudbreak.service.usages

import com.sequenceiq.cloudbreak.domain.CbUsageFilterParameters
import com.sequenceiq.cloudbreak.domain.CloudbreakUsage

interface CloudbreakUsagesRetrievalService {
    fun findUsagesFor(params: CbUsageFilterParameters): List<CloudbreakUsage>
}
